src/Storage/Controllers/DataController.cs

```diff
@@ -287,6 +296,8 @@ public async Task<ActionResult<DataElementList>> GetMany(int instanceOwnerPartyI
 
             DataElement dataElement = await _dataRepository.Create(newData);
             dataElement.SetPlatformSelfLinks(_storageBaseAndHost, instanceOwnerPartyId);
+            
```

Comment:
Should the logic to add the hash to the data element be included here somewhere ?